### PR TITLE
clickhouseexporter: update table schema

### PR DESCRIPTION
**Pull request by hanjm**
**Description:** <Describe what has changed.>

update table schema
**Link to tracking Issue:** <Issue number if applicable>
#8028
**Testing:** <Describe what testing was performed and which tests were added.>
1. unit test.
**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Comment by hanjm** [comment]
hi @dmitryax , Could you help review, Thank you.

**Comment by dmitryax** [comment]
Hey @hanjm can you please break down the PR into several for each self contained change?

**Comment by hanjm** [comment]
@dmitryax I want to split it into two PR, one for update table schema, one for update document, here is the first one, is it OK?

**Comment by dmitryax** [comment]
@hanjm sounds good, thanks. Please rebase and resolve conflicts

**Comment by hanjm** [comment]
@dmitryax rebase done, but ci breaks by other components test.

**Comment by dmitryax** [comment]
@hanjm it doesn't look like it's rebased to the latest main. There are still conflicts

**Comment by hanjm** [comment]
@dmitryax oh, the go.mod file is changed so frequently, I have rebase agagin, thanks.

**Comment by dmitryax** [comment]
@hanjm please fix a failing check

**Comment by hanjm** [comment]
@dmitryax fixed, thanks.

File: exporter/clickhouseexporter/exporter.go [hunk 1]
```diff
@@ -73,27 +74,30 @@ func (e *clickhouseExporter) pushLogsData(ctx context.Context, ld plog.Logs) err
 		defer func() {
 			_ = statement.Close()
 		}()
+		var serviceName string
 		for i := 0; i < ld.ResourceLogs().Len(); i++ {
 			logs := ld.ResourceLogs().At(i)
 			res := logs.Resource()
-			resourceKeys, resourceValues := attributesToSlice(res.Attributes())
+			resAttr := attributesToMap(res.Attributes())
+			if v, ok := res.Attributes().Get(conventions.AttributeServiceName); ok {
```
Comment (dmitryax) [inline exporter/clickhouseexporter/exporter.go:82]: Do you want to keep `service.name` duplicated in the attributes?

**Review by dmitryax — COMMENTED** [review]
Just couple nits. Otherwise LGTM

File: exporter/clickhouseexporter/exporter.go [hunk 2]
```diff
@@ -18,13 +18,14 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
-	"strings"
 	"time"
 
-	_ "github.com/ClickHouse/clickhouse-go" // For register database driver.
```
[the remark this answers is not in the document]
Comment (hanjm) [reply exporter/clickhouseexporter/exporter.go:24]: fixed

File: exporter/clickhouseexporter/exporter.go (hunk 1 above)
Comment (hanjm) [reply exporter/clickhouseexporter/exporter.go:81]: yes, just copy service.name to separate column for primary index, not modify the original attributes.

**Review by dmitryax — APPROVED** [review]

**Pull request merged**